susemanager-utils/testing/docker/master/uyuni-master-base/Dockerfile

```diff
@@ -11,6 +11,7 @@ RUN /root/add_packages.sh
 
 # add schema RPMs for migration tests
 ADD susemanager-schema-4.1.8-1.2.uyuni.noarch.rpm /root/susemanager-schema-4.1.8-1.2.uyuni.noarch.rpm
```

Comment:
Do we need the old one?

Reply:
the schema migration test migrate from 4.1 to 4.2.
We kept it for compatible test as currently we migrate from the same version but with Pg12 which had a different result.